library/CMakeLists.txt

```diff
@@ -233,6 +233,10 @@ IF(UNIX)
   # Don't produce debug info for generated stubs
   SET_SOURCE_FILES_PROPERTIES(DataStatics.cpp DataStaticsCtor.cpp DataStaticsFields.cpp
                               PROPERTIES COMPILE_FLAGS "-g0 -O1")
+
+  # Don't emit thousands of warnings
+  SET_SOURCE_FILES_PROPERTIES(DataStaticsFields.cpp PROPERTIES COMPILE_FLAGS
+    "-Wno-missing-field-initializers -Wno-delete-non-virtual-dtor")
```

Comment:
Have you checked (e.g. with `make VERBOSE=1`) to make sure this doesn't overwrite the `-g0 -O1` from above?